examples/minimal/packages/contracts/mud.config.ts

```diff
@@ -1,7 +1,8 @@
-import { mudConfig } from "@latticexyz/world/register";
+import { mudConfig } from "@latticexyz/world/snapSync";
 import { resolveTableId } from "@latticexyz/config";
 
 export default mudConfig({
+  snapSync: true,
```

Comment:
just as a general note, I think this is the first time the mud config has had an option that could effect network code

since we're passing in the mud config to the network config, maybe should just use this key to determine if snap sync should be used?